perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -865,6 +874,11 @@ sub construct_bin_report_form {
 sub report : Chained('property') : Args(0) {
     my ($self, $c) = @_;
 
+    $c->stash->{original_booking_report}
+        = FixMyStreet::DB->resultset("Problem")
+        ->find( { id => $c->get_param('original_booking_id') } )
+        if $c->get_param('original_booking_id');
```

Comment:
I guess doing this means the `$booking_report` look up in `waste_munge_report_data` in SLWP2 (and Brent, no need to do Merton as they're moving to SLWP2 shortly) isn't needed, it can use this stage variable?

Reply:
True; that is where I copied the DB lookup from.